convert2rhel/pkghandler.py

```diff
@@ -72,29 +73,42 @@ def call_yum_cmd(command, args="", enablerepo=None, disablerepo=None,
     """Call yum command and optionally print its output."""
     loggerinst = logging.getLogger(__name__)
 
-    cmd = "yum %s -y" % (command)
+    cmd = "yum %s -y" % command
     # disablerepo parameter must be added before the enablerepo parameter
 
     if disablerepo is None:
         disablerepo = []
     if disablerepo:
         repos = disablerepo
-    else:
+    elif tool_opts.disablerepo:
         repos = tool_opts.disablerepo
```

Comment:
Do we have a test for this branch?

Reply:
Added here: https://github.com/oamg/convert2rhel/pull/84/files#diff-cdec7561cf20f5b5b314293008180b42R83